### PR TITLE
add ios blog post

### PR DESCRIPTION
**Pull request by r10s**
closes #408 

please merge only when 1.16 is actually available in the app store (it is on its way :)

### CODE REVIEW

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: iOS is catching up!
```
Comment (hpk42) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion
title: Delta Chat iOS is catching up!
```

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+image: ../assets/blog/2021-02-04-context-menu-small.jpg
+---
+
+_Delta Chat iOS_ is available
+[since about a year now](2020-01-09-iOS-appstore-release) 🎉 —  
+while being nicely usable and working
+(all platforms share the same,
+stable [Rust-core](2019-05-08-xyiv#the-coming-delta-chat-rustocalypse)),
+compared to the more mature Android and Desktop versions,
+some UI-features were just missing.
```
Comment (hpk42) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion
while being nicely usable and working,
compared to the more mature Android and Desktop versions,
some UI-features were just missing from our iOS app.
Note that all platforms share the same,
stable [Rust-core](https://github.com/deltachat/deltachat-core-rust) so you can expect the same core functionality and stability that our longer evolving Android and Desktop apps enjoy.
```

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  we added **global search**, **disappearing messages**, **swipe-to-reply**,
+  improved **video recording** and **message view**
+  and [much more](https://github.com/deltachat/deltachat-ios/blob/master/CHANGELOG.md#delta-chat-ios-changelog).
+
+**Currently, we are working on improving background notifications** -
+depending how often you use Delta Chat
+and depending on how many other apps you have installed,
+iOS does not allow us to check for new messages as we would like to do.
+This is a known issue,
+however, we have some ideas how that could be targeted - stay tuned.
```
Comment (hpk42) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion
This is a known issue and there are ongoing efforts to address the issue -- stay tuned :)
```

File: _posts/2021-02-04-ios-is-catching-up.md (hunk 1 above)
Comment (r10s) [reply _posts/2021-02-04-ios-is-catching-up.md]: makes sense :)

**Review by hpk42 — APPROVED** [review]
very nice, just a few nitpicks and fixes here and there.

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+(maybe beside Apple's :)
+
+
+
+## Try out the new iOS releases!
+
+**As always, you'll find the new iOS version in Apple's
+[App Store](https://apps.apple.com/us/app/delta-chat/id1459523234).**
+
+You are interesting in testing new features?
```
Comment (hpk42) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion
Are you maybe interested in testing new features?
```

**Review by adbenitez — APPROVED** [review]

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+and they are a good example
+why native UI development rules for every-day-apps as messengers -
+they just _look, feel and work better_ than platform-independent compromises.
+
+![Screenshots of iOS context menus](../assets/blog/2021-02-04-context-menu.png)
+
+
+### Send images with text and edit images
+
+- **Before sending an image**, you can now **add some text:**
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md:46]: maybe add "gif or video" here

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+  you can even **draw on it and do some basic editing:**
+  Tap the image selected to send
+  and then the **edit icon** in the upper right corner.
+  
+![Screenshots of iOS image sending options](../assets/blog/2021-02-04-edit-image.png)
+
+### What else?
+
+- We did some effort to **improve accessibility** -
+  thanks for blind and other people helping us to target some issues.
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md]: I don't know if our helper is blind actually. Maybe just say thanks "to our comminity helping us to target some issues"

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+iOS does not allow us to check for new messages as we would like to do.
+This is a known issue and there are ongoing efforts to address the issue -- stay tuned :)
+
+Finally, it is worth mentioning, that, despite all these new features
+the **minimal requirements have not changed:**
+iOS 11, iPhone 5s or iPad 5/Air/Mini are still sufficient.  
+Although supporting old systems is quite some additional work,
+we have resisted the temptation to increase requirements several times.
+So, existing devices can be continued for everyone's benefit
+(maybe beside Apple's :)
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md:88]: 🤣

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+  the image will be shown above the input field where you can add some text
+  and finally tap the send-button.
+
+- In the same way, you can **add some description to documents or files**
+
+- Also, before sending an image,
+  you can even **draw on it and do some basic editing:**
+  Tap the image selected to send
+  and then the **edit icon** in the upper right corner.
+
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md]: maybe we can mention here as well that you can preview drafted documents? I'm not sure it it's that important. But that's part of the new feature, too

File: _posts/2021-02-04-ios-is-catching-up.md (hunk 5 above)
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md:46]:
```suggestion
- **Before sending an image, gif or video**, you can now **add some text:**
```

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+### Send images with text and edit images
+
+- **Before sending an image**, you can now **add some text:**
+  Use the camera or the gallery as usual - 
+  the image will be shown above the input field where you can add some text
+  and finally tap the send-button.
+
+- In the same way, you can **add some description to documents or files**
+
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md:62]:
```suggestion

Moreover you can have a closer look at a draft by tapping on it. A large preview opens which is especially useful for documents to double check the file you are about to send. 
The preview allows even basic editing, like **drawing on images** or **cutting videos**. To enable editing you find a **edit button** in the upper right corner.
```

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+  you can even **draw on it and do some basic editing:**
+  Tap the image selected to send
+  and then the **edit icon** in the upper right corner.
+  
+![Screenshots of iOS image sending options](../assets/blog/2021-02-04-edit-image.png)
+
+### What else?
+
+- We did some effort to **improve accessibility** -
+  thanks for blind and other people helping us to target some issues.
```
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion
  thanks to our community helping us to target some issues.
```

File: _posts/2021-02-04-ios-is-catching-up.md (hunk 8 above)
Comment (cyBerta) [inline _posts/2021-02-04-ios-is-catching-up.md]:
```suggestion  
```

File: _posts/2021-02-04-ios-is-catching-up.md (hunk 5 above)
Comment (r10s) [reply _posts/2021-02-04-ios-is-catching-up.md:46]: i'll add that to the paragraph below.

File: _posts/2021-02-04-ios-is-catching-up.md (hunk 9 above)
Comment (r10s) [reply _posts/2021-02-04-ios-is-catching-up.md:62]: thanks or pointing the video-cutting out, this is major, indeed - i forgot about that, sorry!

**Review by cyBerta — APPROVED** [review]

File: _posts/2021-02-04-ios-is-catching-up.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+iOS does not allow us to check for new messages as we would like to do.
+This is a known issue and there are ongoing efforts to address the issue -- stay tuned :)
+
+Finally, it is worth mentioning, that, despite all these new features
+the **minimal requirements have not changed:**
+iOS 11, iPhone 5s or iPad 5/Air/Mini are still sufficient.  
+Although supporting old systems is quite some additional work,
+we have resisted the temptation to increase requirements several times.
+So, existing devices can be continued for everyone's benefit
+(maybe beside Apple's :)
```
Comment (Simon-Laux) [inline _posts/2021-02-04-ios-is-catching-up.md:88]: syntactically there is a closing parenthesis missing:
```suggestion
(maybe beside Apple's :))
```
and if that doesn't look as good maybe replace the emoticon with an emoji:
😄 or 😏
```suggestion
(maybe beside Apple's 😄)
```
```suggestion
(maybe beside Apple's 😏)
```

Comment (r10s) [reply _posts/2021-02-04-ios-is-catching-up.md:88]: hm, i personally like the original, syntactically incorrect variant most (i read it as an orthographic ligature :) (ahh, and there is https://xkcd.com/541/ :)

**Pull request merged**